src/datastore/postgres-store.ts

```diff
@@ -690,6 +763,34 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
       updatedEntities.markedNonCanonical.smartContracts += smartContractResult.rowCount;
     }
 
+    const nameResult = await client.query(
```

Comment:
Looks like we're missing re-org handling for the `subdomains` table, seems like it should be handled here just like the `names` table?